docs/source/notebooks/Full_process_example_1.ipynb

```diff
@@ -262,9 +315,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/david/.conda/envs/birdy/lib/python3.7/site-packages/ipykernel_launcher.py:14: FionaDeprecationWarning: Collection.__next__() is buggy and will be removed in Fiona 2.0. Switch to `next(iter(collection))`.\n",
```

Comment:
This is easily fixable.

Reply:
Would be nice to remove warnings if possible without too much fuss, but it was not clear to me where this was coming from.

Reply:
I think:

```py
shdf = [vector.next()["geometry"]]
```
must simply become:
```py
shdf = [next(iter(vector))["geometry"]]
```